config/env/env_types.go

```diff
@@ -0,0 +1,161 @@
+package env
+
+import (
+	"errors"
+	"os"
+	"regexp"
+	"strconv"
+)
+
+var TypeError = errors.New("type could not be converted")
+var EnvironmentError = errors.New("environment variable not found")
```

Comment:
This error could communicate more to help the user. i.e. "environment variable 'FOO' not set"